packages/IntegrationTest/windows/integrationtest/TestHostHarness.h

```diff
@@ -46,17 +46,20 @@ class TestHostHarness : public winrt::implements<TestHostHarness, winrt::Windows
 class TestHostHarnessRedboxHandler
     : public winrt::implements<TestHostHarnessRedboxHandler, winrt::Microsoft::ReactNative::IRedBoxHandler> {
  public:
-  TestHostHarnessRedboxHandler(winrt::weak_ref<TestHostHarness> &&weakHarness)
+  TestHostHarnessRedboxHandler(winrt::weak_ref<TestHostHarness> &&weakHarness) noexcept
       : m_weakHarness(std::move(weakHarness)) {}
 
   void ShowNewError(
-      winrt::Microsoft::ReactNative::IRedBoxErrorInfo info,
-      winrt::Microsoft::ReactNative::RedBoxErrorType type);
-  bool IsDevSupportEnabled();
-  void UpdateError(winrt::Microsoft::ReactNative::IRedBoxErrorInfo /*info*/);
-  void DismissRedBox();
+      const winrt::Microsoft::ReactNative::IRedBoxErrorInfo &info,
```

Comment:
nit: Indent seems to be one level off

Reply:
This is how Clang format decided it wanted to format this for some reason 🤷‍♀️.